packages/design-system-docs/src/pages/components/Tooltip/Tooltip.example.jsx

```diff
@@ -5,90 +5,97 @@ import ReactDOM from 'react-dom';
 ReactDOM.render(
   <>
     <div className="ds-u-display--flex ds-u-align-items--center ds-u-margin-y--2">
-      <p className="ds-u-margin--0">Tooltip using an icon trigger</p>
+      <p className="ds-u-margin--0">Tooltip with icon trigger</p>
       <Tooltip
         ariaLabel="Label describing the subject of the tooltip"
-        triggerContent={<TooltipIcon />}
-        triggerClassName="ds-c-tooltip__trigger-icon"
-        triggerActiveClassName="ds-c-tooltip-icon--active"
+        className="ds-c-tooltip__trigger-icon"
+        activeClassName="ds-c-tooltip-icon--active"
+        title="Tooltip trigger uses <TooltipIcon> for the trigger content"
       >
-        {'Tooltip trigger uses <TooltipIcon> for the trigger content'}
+        <TooltipIcon />
       </Tooltip>
     </div>
     <div className="ds-u-display--flex ds-u-align-items--center ds-u-margin-y--2">
-      <p className="ds-u-margin--0 ds-u-margin-right--1">Tooltip using a</p>
-      <Tooltip triggerContent="text trigger" triggerClassName="ds-c-tooltip__trigger-link">
-        Tooltip trigger is styled with dashed underline
-      </Tooltip>
+      <p className="ds-u-margin--0">
+        {'Tooltip with '}
+        <Tooltip
+          className="ds-c-tooltip__trigger-link"
+          component="a"
+          title="Tooltip trigger uses <a> for the trigger, styled with dashed underline"
```

Comment:
Ah thats a good point, so the default styling for links across HCgov uses `border-bottom` instead of `text-decoration`? 

I think we'd want to keep it consistent between tooltips and links, and since the CMSDS link styling uses `text-decoration`, I will probably keep it the same for `ds-c-tooltip__trigger-link`. That being said, if HCgov uses `border-bottom` consistently, we could always override `ds-c-tooltip__trigger-link` and default link styles in the HCgov DS

Reply:
It's strange how much thicker and clunkier dashed is for text-decoration vs. what Louis shows above with border-bottom-style. It almost looks closer to 'dotted' than 'dashed'

Reply:
yeah thats true. I guess if no one is going to use the `text-decoration` style i'd be fine with going with `border-bottom-style`, but it would be great to check with a designer on this

Reply:
> Ah thats a good point, so the default styling for links across HCgov uses `border-bottom` instead of `text-decoration`?

Correct, but I'm not tied to the `border` solution here; I actually think using `text-decoration` (perhaps with `text-underline-offset`) is much more semantically correct.  HCGov's styling is really out-of-date and shouldn't be used as the model in 99% of circumstances.

I mainly wanted to pass this on since Kristen brought up that the dashed underline using `text-decoration` made the text more difficult to read, which I think is a fair criticism.

Anyway, +1 to getting a designer involved here; ideally I think all of our link underlines should have similar placement regardless of how they're being used.